### ISSUE TITLE
Add Distance Map use-case for Brightness Temperature

### ISSUE THREAD

**Issue by TaraJensen** [dtcenter/METplus#642]
*Replace italics below with details for this issue.*

## Describe the New Feature ##
Part of the UW-CIMSS project includes developing a use-case for the evaluation brightness temperature to compute Distance Map metrics.  Sarah Griffin has provided sample data and the Grid-Stat configuration she used for the initial analysis.  They can be found on kiowa:/d1/personal/jensen/JTTI-UW

### Acceptance Testing ###
OU-CAPS ensemble members and Satellite brightness temperature originally from:
Now found at kiowa:/d1/personal/jensen/JTTI_UW
New functionality:  Need to sort out how to read in UW_CIMSS netCDF format.  

Running MET independently using the original Grid_Stat config used by UW_CIMSS and commands generated by METplus works.
Running MET through METplus using the original Grid_Stat config used by UW_CIMSS works
Running MET through METplus using a slightly modified Grid_Stat config with 1 env variable passed from METplus .conf to MET config does not work.

### Time Estimate ###
1 day to get working
1-2 day to add use-case

### Sub-Issues ###
Consider breaking the new feature down into sub-issues.
- [x ] Get Filename template working
- [ ] Wrap MET config file
- [ ] Define sample data tarball
- [ ] Add use-case to CAM application directory 

### Relevant Deadlines ###
November 2020

### Funding Source ###
2770061

### Assignee ###
- [x ] Select **engineer(s)** or **no engineer** required
- [ x ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [ ] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## New Feature Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by georgemccabe** [comment]
This use case was merged into develop, but there is another pull request to handle updates to the CI to properly handle getting the new truth data generated. Once that is merged in, we should create a pull request to update develop-ref from develop and can close this issue.

**Comment by georgemccabe** [comment]
PR #842 updates develop-ref to generate new output data volumes for testing. This issue is complete.
